mappings/net/minecraft/client/particle/ParticleLavaEmber.mapping

```diff
@@ -1,7 +1,7 @@
-CLASS none/bni net/minecraft/client/particle/ParticleLava
+CLASS none/bni net/minecraft/client/particle/ParticleLavaEmber
```

Comment:
I've grouped water related things as `WaterSomething` so this seemed to make sense to me.
(This should be the particle that shoots out of lava in an arc and spawns smoke particles as it goes.)